### PR TITLE
Rename nessie-quarkus-cli to nessie-server-admin-tool

### LINKED ISSUE #8481: Rename nessie-quarkus-cli
### Issue description

See #8245 fro context. We should rename `nessie-quarkus-cli` because the name is misleading (users might mix it up with pynessie or the new CLI/REPL tool.)

### PR DESCRIPTION
**Pull request by dimas-b**
Rename nessie-quarkus-cli to nessie-server-admin-tool
    
Also move code under ./tools/server-admin
    
This is a follow-up to #8245
Closes #8481

### CODE REVIEW

File: helm/nessie/README.md [hunk 1]
```diff
@@ -54,75 +54,75 @@ $ helm uninstall --namespace nessie-ns nessie
...
-| bigtable.appProfileId | string | `"default"` | The Google Cloud Bigtable app profile ID. |
-| bigtable.instanceId | string | `"nessie-bigtable"` | The Google Cloud Bigtable instance ID. |
-| bigtable.projectId | string | `"my-gcp-project"` | The Google Cloud project ID. |
+| Key | Type | Default | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```
Comment (snazy) [inline helm/nessie/README.md:57]: unrelated big diff

File: site/docs/downloads/index.md [hunk 2]
```diff
@@ -77,9 +77,9 @@ java -jar https://github.com/projectnessie/nessie/releases/download/nessie-{{ ve
 Requires Java 17 or newer.
 
 ```bash
-curl -o nessie-quarkus-cli-{{ versions.nessie }}-runner.jar \
-  https://github.com/projectnessie/nessie/releases/download/nessie-{{ versions.nessie }}/nessie-quarkus-cli-{{ versions.nessie }}-runner.jar
-java -jar nessie-quarkus-cli-{{ versions.nessie }}-runner.jar
+curl -o nessie-admin-cli-{{ versions.nessie }}-runner.jar \
```
Comment (snazy) [inline site/docs/downloads/index.md:80]: This change needs to go into a separate PR (similar to #8468)

File: site/docs/guides/migration.md [hunk 3]
```diff
@@ -76,11 +76,11 @@ Exported Nessie repository, 100 commits into 1 files, 1 named references into 1
 Once the
 export process has completed, the ZIP file will be available at `/tmp/export-<date>.zip`.
 
-Download the `nessie-quarkus-cli-x.y.z-runner.jar` corresponding to the **new** Nessie version, here
-we assume version 0.75.0:
+Download the `nessie-admin-cli-x.y.z-runner.jar` corresponding to the **new** Nessie version, here
```
Comment (snazy) [inline site/docs/guides/migration.md:79]: Changes in this file need to go into a separate PR (similar to https://github.com/projectnessie/nessie/pull/8468)

File: site/docs/guides/migration.md [hunk 4]
```diff
@@ -76,11 +76,11 @@ Exported Nessie repository, 100 commits into 1 files, 1 named references into 1
...
+we assume version 0.83.0:
```
Comment (snazy) [inline site/docs/guides/migration.md:80]: Maybe add a note for older versions

File: site/in-dev/export_import.md [hunk 5]
```diff
@@ -15,26 +15,26 @@ such as:
 ## Usage
 
 The Nessie Quarkus CLI too requires direct access to the database used by Nessie. The executable is
-named `nessie-quarkus-cli-x.y.z-runner.jar` and can be downloaded from the
+named `nessie-admin-cli-x.y.z-runner.jar` and can be downloaded from the
 [release page on GitHub](https://github.com/projectnessie/nessie/releases) and is available
-for Nessie 0.43.0 or newer.
+for Nessie 0.83.0 or newer.
```
Comment (snazy) [inline site/in-dev/export_import.md:20]: Similar, a note for 0.43..0.82

Comment (snazy) [reply site/in-dev/export_import.md:20]: Actually, no need to mention a verison here - this page is per release-version

**Review by snazy — COMMENTED** [review]
Need to update the release CI workflows

**Comment by adutra** [comment]
Should we remove the `-cli` suffix? E.g. `nessie-server-admin` or `nessie-admin-tool`.

**Comment by snazy** [comment]
> Should we remove the `-cli` suffix? E.g. `nessie-server-admin` or `nessie-admin-tool`.

nessie-server-admin-tool sounds fine

File: helm/nessie/README.md (hunk 1 above)
Comment (dimas-b) [reply helm/nessie/README.md:57]: IJ auto-formats :shrug:

File: site/in-dev/export_import.md (hunk 5 above)
Comment (dimas-b) [reply site/in-dev/export_import.md]: updated

File: helm/nessie/README.md (hunk 1 above)
Comment (dimas-b) [reply helm/nessie/README.md:57]: fixed

File: site/docs/guides/migration.md (hunk 4 above)
Comment (dimas-b) [reply site/docs/guides/migration.md]: Is it really worth the trouble?

File: site/docs/guides/migration.md (hunk 3 above)
Comment (dimas-b) [reply site/docs/guides/migration.md]: #8484

File: site/docs/guides/migration.md (hunk 4 above)
Comment (dimas-b) [reply site/docs/guides/migration.md]: #8484

File: site/docs/downloads/index.md (hunk 2 above)
Comment (dimas-b) [reply site/docs/downloads/index.md]: #8484

File: site/in-dev/export_import.md [hunk 6]
```diff
@@ -15,26 +15,25 @@ such as:
 ## Usage
 
 The Nessie Quarkus CLI too requires direct access to the database used by Nessie. The executable is
```
Comment (adutra) [inline site/in-dev/export_import.md:17]: This paragraph and the previous section both need some rephrasing. I also wonder if the page title shouldn't be changed to "Nessie Server Admin Tool".

Comment (dimas-b) [reply site/in-dev/export_import.md]: good point. updated

**Comment by snazy** [comment]
> Need to update the release CI workflows

Still find occurences of `nessie-quarkus-cli` in release-publish.yml, which will break releases

**Comment by dimas-b** [comment]
`rg` does not look into hiden dirs :facepalm:

File: site/docs/downloads/index.md [hunk 7]
```diff
@@ -77,9 +77,9 @@ java -jar https://github.com/projectnessie/nessie/releases/download/nessie-{{ ve
 Requires Java 17 or newer.
 
 ```bash
-curl -o nessie-quarkus-cli-{{ versions.nessie }}-runner.jar \
-  https://github.com/projectnessie/nessie/releases/download/nessie-{{ versions.nessie }}/nessie-quarkus-cli-{{ versions.nessie }}-runner.jar
-java -jar nessie-quarkus-cli-{{ versions.nessie }}-runner.jar
+curl -o nessie-server-admin-tool-{{ versions.nessie }}-runner.jar \
```
Comment (snazy) [inline site/docs/downloads/index.md:80]: This change must also go into the follow-up PR

**Review by snazy — COMMENTED** [review]
LGTM - except one must-revert/move

File: site/docs/downloads/index.md (hunk 7 above)
Comment (dimas-b) [reply site/docs/downloads/index.md]: It's already in #8484. I've removed this change from this PR.

**Comment by snazy** [comment]
Took the freedom to fix the merge conflict and force-pushed

**Review by snazy — APPROVED** [review]
